tools/fpgabist/bist_dma.py

```diff
@@ -38,9 +38,10 @@ def __init__(self):
         self.executables = {'fpga_dma_test': '0'}
 
     def run(self, gbs_path, bus_num):
-        bc.load_gbs(gbs_path, bus_num)
+        bc.load_gbs(self, gbs_path, bus_num)
```

Comment:
It doesn't need to use `self`. You can just pass `cwd` to function. Or, in your case: `self.cwd`.

Reply:
I agree just use `self.cwd`